### PR TITLE
merge feature/gefs_v12_3 into ops for GEFSv12.3.0

### PR DESCRIPTION
**Pull request by XianwuXue-NOAA**

### CODE REVIEW

File: sorc/checkout.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ if [[ ! -d global-workflow.fd ]] ; then
     rm -f ${logs_dir}/checkout-global-workflow.log
     git clone https://github.com/NOAA-EMC/global-workflow.git global-workflow.fd >>  ${logs_dir}/checkout-global-workflow.log 2>&1
     cd global-workflow.fd
-    git checkout gefs_v12.2.0-1
+    git checkout release/gefs_v12 #gefs_v12.2.0-1
```
Comment (XianwuXue-NOAA) [inline sorc/checkout.sh]: Will change the branch name to tag when the tag is available.

Comment (lipan-NOAA) [reply sorc/checkout.sh]: @XianwuXue-NOAA I agree

**Review by lipan-NOAA — APPROVED** [review]

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Review by lipan-NOAA — APPROVED** [review]

**Comment by XianwuXue-NOAA** [comment]
@WalterKolczynski-NOAA could you create a tag (gefs_v12.3.0-0?) in global-workflow for branch "release/gefs_v12" since Li and I have tested successfully and fixed the UPP issue and updated the UPP tag.

**Comment by WalterKolczynski-NOAA** [comment]
Tag created: https://github.com/NOAA-EMC/global-workflow/releases/tag/gefs_v12.3.0-0

**Comment by XianwuXue-NOAA** [comment]
Thanks @WalterKolczynski-NOAA

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**